lib/matplotlib/tests/test_gridspec.py

```diff
@@ -1,5 +1,5 @@
 import matplotlib.gridspec as gridspec
-from nose.tools import assert_raises, assert_equal
+from numpy.testing import assert_raises, assert_equal
```

Comment:
So I've been testing it out without nose installed and `numpy.testing.assert_raises` doesn't work with nose installed. Can you replace it with the `pytest.raises` context manager?